eslint.config.mjs

```diff
@@ -0,0 +1,45 @@
+import { defineConfig } from 'eslint/config';
+import globals from 'globals';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import js from '@eslint/js';
+import { FlatCompat } from '@eslint/eslintrc';
+import { fixupConfigRules } from '@eslint/compat';
+import reactDom from 'eslint-plugin-react-dom';
+import reactWebApi from 'eslint-plugin-react-web-api';
+import reactHooksExtra from 'eslint-plugin-react-hooks-extra';
+import reactNamingConvention from 'eslint-plugin-react-naming-convention';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+});
+
+export default defineConfig([
+  {
+    extends: fixupConfigRules(compat.extends('satya164')),
+    plugins: {
+      '@eslint-react/dom': reactDom,
+      '@eslint-react/hooks-extra': reactHooksExtra,
+      '@eslint-react/naming-convention': reactNamingConvention,
+      '@eslint-react/web-api': reactWebApi,
+    },
```

Comment:
This project's configuration extends [satya164](https://github.com/satya164/eslint-config-satya164) which I think is still using the old config format. ESLint complained that it couldn't find some `@eslint-react/*` plugins (even though they were there) and I had to manually add them to the config to make it work.